Magento_Checkout/web/template/summary/cart-items.html

```diff
@@ -7,7 +7,7 @@
     data-bind="
         mageInit: {
             'collapsible': {
-                'openedState': 'active', 'active': isItemsBlockExpanded()
+                'openedState': 'active', 'active': true
```

Comment:
please improve formatting - each option should be in a separate line
```suggestion
                'openedState': 'active', 
                'active': true
```